build-system/server.js

```diff
@@ -646,12 +646,12 @@ app.get(['/dist/cache-sw.min.html', '/dist/cache-sw.max.html'], function(req, re
  */
 function replaceUrls(mode, file) {
   if (mode) {
-    file = file.replace('https://cdn.ampproject.org/viewer/google/v5.js', 'https://cdn1.ampproject.org/viewer/google/v5.js');
```

Comment:
ha, the code was interesting :-) but i guess removing them will break something?